composer.lock

```diff
@@ -538,6 +538,70 @@
         }
     ],
     "packages-dev": [
+        {
```

Comment:
are we just updating transitive dependency versions or did you forget to include a change to `compose.json` in this review?